RELEASE_NOTES.md

```diff
@@ -8,7 +8,7 @@ Details about the thing that changed that needs to get included in the Release N
 -->
 
 
-# Windows azure-chef-extension 1210.12.109.1000 release notes:
+# Windows azure-chef-extension 1210.12.109.1002 release notes:
 In this release, we have added support for environment variables in windows chef extension.
 

```

Comment:
Notes need to be modified here. We should mention about the bug fixes here and remove the environment variables content.